docs/registry/configure.md

```diff
@@ -351,13 +313,15 @@ When a module is instantiated, it has access to the following default environmen
 | `VIAM_MODULE_DATA` | A persistent folder location a module can use to store data across reboots and versions. This location is a good place to store [python virtual environments](/sdks/python/python-venv/).<br>Example: `$VIAM_HOME/module-data/cloud-machine-id/my-module-name/` |
 | `VIAM_MODULE_ID` | The module ID of the module. <br>Example: `viam:realsense` |
 
+{{< /expand >}}
+
 ## Local modules
 
 If you wish to add a module to your machine without uploading it to the Viam registry, you can add your module as a _local module_.
```

Comment:
so this is weird here. If this page was called modules it would make sense. This either indicates we should have this page be named Modules, or modules and modular resources
It could also indicate this info needs to move. I think make a note of it and let's worry about it later once the rest of the content feels like it's in the right place.